### PR TITLE
Add ci/cd to frontend

### PR DESCRIPTION
**Pull request by szymekc**

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ services:
       - sigmaChat
     env_file:
       - .env.production
-
```
Comment (MacenDi) [inline compose.yaml]: Plz remove this change :////

Comment (szymekc) [reply compose.yaml]: Done :) :)

Comment (szymekc) [reply compose.yaml]: I left this one in as an exercise for the reviewer btw

**Comment by MacenDi** [comment]
LGTM ;]

image

File: compose.prod.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - AUTH0_SECRET=${AUTH0_SECRET}
+      - AUTH0_SESSION_ABSOLUTE_DURATION=${AUTH0_SESSION_ABSOLUTE_DURATION}
+      - NEXT_PUBLIC_API_HUB=${NEXT_PUBLIC_API_HUB}
+      - NEXT_PUBLIC_API_URL=${NEXT_PUBLIC_API_URL}
+      - NEXT_PUBLIC_UNTHINKABLE=${NEXT_PUBLIC_UNTHINKABLE}
+networks:
+  sigmaChat:
+    name: sigmaChat
+
+
```
Comment (senbar) [inline compose.prod.yaml]: ???

File: compose.prod.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (senbar) [inline compose.prod.yaml]: empty lines hello?

File: compose.prod.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (senbar) [inline compose.prod.yaml]: syling error

**Review by senbar — CHANGES_REQUESTED** [review]
image

File: compose.prod.yaml (hunk 4 above)
Comment (szymekc) [reply compose.prod.yaml]: Fixed

File: compose.prod.yaml (hunk 3 above)
Comment (szymekc) [reply compose.prod.yaml]: Fixed

File: compose.prod.yaml (hunk 2 above)
Comment (szymekc) [reply compose.prod.yaml]: Fixed

**Pull request merged**